### DISCUSSION TITLE
How to provide workspaces with raw files?

### DISCUSSION THREAD

**Discussion by redxtech** [Raytwo/ARCropolis#521] [General]
Heyo!

I'm in the process of making what is effectively a package that can be created and then written to an SD card that can be put into a switch, with everything working.

Something I'm wanting to include in this would be pre-defined workspaces for the mod manager. I've looked at the files on my switch after making some workspaces, but it appears as though some of the files are in unexpected locations & some functionality is unclear, so I'm just wanting to confirm that the following configuration would work when placed on a fresh switch.

**Goals:**
Two pre-defined profiles, one with a single mod enabled (default), and one for HDR, with only the 3 HDR mods enabled.

**Current files:**
Config directory: `/ultimate/arcropolis/config/1154720435214108633/7988075541687562371/` - does this directory remain the same on every switch, or should I give it a different name in the prepared package?

`workspace`:
```
 Default
```

`workspace_list`:
```
{"Default":"presets","HDR":"HDR_preset2"}
```
`presets`: (how are the identifiers generated? is it a hash of the folder name? of the folder contents? how can i ensure this stays consistent?)
```
[131143154329]
```
`HDR_preset2`: (same quesion as above)
```
[123019169574,124334938316,94293926859]
```

Am I missing anything, or should this work fine?

**Comment by jozz024** [comment]
hi, those numbers are the users uids, so they will not remain the same on every switch.

**Reply by redxtech** [reply]
> hi, those numbers are the users uids, so they will not remain the same on every switch.

Hmm, is there any way I can structure folders and files that would allow me to share profiles between switches?
